src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -311,6 +315,7 @@ def _run_dace_cpu(program: itir.FencilDefinition, *args, **kwargs) -> None:
         **kwargs,
         build_cache=_build_cache_cpu,
         build_type=_build_type,
+        build_args=_cpu_args,
```

Comment:
I propose to move the definition of `_cpu_args` inside this function (`_run_dace_cpu`).

I also propose this modification to its definition:
```
compiler_args = dace.config.Config.get("compiler", "cpu", "args")
# disable finite-math-only in order to support isfinite/isinf/isnan builtins
if "-ffast-math" in compiler_args:
    compiler_args += " -fno-finite-math-only"
if "-ffinite-math-only" in compiler_args:
    compiler_args.replace("-ffinite-math-only", "")
```